api/src/main/java/edu/cornell/mannlib/vitro/webapp/web/templatemodels/individual/FauxDataPropertyWrapper.java

```diff
@@ -0,0 +1,329 @@
+/* $This file is distributed under the terms of the license in LICENSE$ */
```

Comment:
I have noticed that we have a lot of java classes with this line at the beginning, but do we need that or it is included in the definition of the license

Reply:
Looks to be a JavaDocs license header to not repeat license text in all files. Here are some links trying to discover the origin of these comments:

https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html#packagecomments
https://mavenlibs.com/javadoc/api/org.sonarsource.license-headers/license-headers

Reply:
We can consider how to resolve this for the complete project in the separate thread. I will open an issue for this, we can discuss that and try to improve. Thanks